composer.json

```diff
@@ -21,7 +21,7 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0 || ~8.2.0",
```

Comment:
Please drop support for PHP 7 and also add `config.platform.php` with "8.0.99"

See: https://github.com/laminas/laminas-validator/pull/156/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34